### ISSUE TITLE
Hello bro! I have an hp AIO i5-8250U kaby lake r that i need the efi for paste it to ventura usb that i made is possible to help me?

### ISSUE THREAD

**Issue by uno-bianca** [sealfx/HP-Pavilion-24-b212d-Z8G27AA-AKL-AIO#1]

**Comment by sealfx** [comment]
I have an new EFI hp AIO for Catalina and Monterey if U test boot pass. i think U can Boot it to ventura. [https://drive.google.com/file/d/1rSeR7RFQH-qhLwxk0XtQpsd2_L1qduWm/view?usp=sharing](url)

**Comment by uno-bianca** [comment]
Is not a download link bro! I cant download!

**Comment by sealfx** [comment]
U can hilight full url copy and paste in browser bro!
or
https://github.com/sealfx/HP-Pavilion-24-b212d-Z8G27AA-AKL-AIO/blob/master/EFI_OC_Any.zip

**Comment by uno-bianca** [comment]
Yes now it works bro thx! And somthing else my AIO hp is like in the picture, you think it will work, because your aio is 7th generation and not like this model??!![image](https://github.com/sealfx/HP-Pavilion-24-b212d-Z8G27AA-AKL-AIO/assets/134656150/b3372333-91ed-419f-9086-ef57ea307d74)

**Comment by uno-bianca** [comment]
Dont boot bro! https://github.com/sealfx/HP-Pavilion-24-b212d-Z8G27AA-AKL-AIO/assets/134656150/8c0da235-5d70-46e5-8d12-5a6665ff8492

**Comment by sealfx** [comment]
I not try to use for Ventura if i test again and it work i will tell you Bro.

**Comment by uno-bianca** [comment]
i understand bro

**Comment by uno-bianca** [comment]
![image](https://github.com/sealfx/HP-Pavilion-24-b212d-Z8G27AA-AKL-AIO/assets/134656150/eaa3b7bd-11ac-4a9f-b61b-7e1c4d3d2465)

**Comment by uno-bianca** [comment]
Hello bro sorry for disturbing you i made montery it start but i stuck in the middle and restart itself! I think is a problem with EFI 😔😔
